solidity/scripts/misbehaved-groups.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Important: the KeepRandomBeaconOperatorRewardsStub
+ * and the TokenStakingSlashingStubs contract should be deployed to the network!
+ * 
+ * This script:
+ *  - emits fake events (UnauthorizedSigningReported, RelayEntryTimeoutReported)
+ *  */ 
+
+// The KeepRandomBeaconOperatorRewardsStub contract should be deployed to the network
+const Operator = artifacts.require("../stubs/KeepRandomBeaconOperatorRewardsStub.sol");
+const crypto = require("crypto")
+
+function getAccounts() {
+  return new Promise((resolve, reject) => {
+      web3.eth.getAccounts((error, accounts) => {
+        resolve(accounts);
+      });
+  });
+}
+
+async function registerNewGroup (keepRandomBeaconOerator, accounts) {
+  const groupPubKey = crypto.randomBytes(32);
+  await keepRandomBeaconOerator.registerNewGroup(groupPubKey)
+  await keepRandomBeaconOerator.setGroupMembers(groupPubKey, [accounts[1], accounts[2], accounts[1]]);
```

Comment:
Is it intentional to put account 1 twice?

Reply:
Yep. For testing when a given operator is in a given group many times.